__tests__/body-customisation.test.ts

```diff
@@ -0,0 +1,25 @@
+import { setBodyParameters} from '../src/automation'
+import { BodyParam} from '../src/types/index'
+import { sampleMessages } from '../src/utils/setupTests'
+import typeorm = require('typeorm')
+
+describe('Unit Tests', () => {
+
+  test('Test setBodyParameters with 2 parameters set', async () => {
+    const body = "<h1>PARAMS_Lastname PARAMS_Firstname Send automated messages with Candymail</h1><p>Now with HTML support</p><a href='https://saasbase.dev/candymail'>Learn more here</a>"
+    const params=[{key:"Firstname",value:"Harry"},{key:"Lastname",value:"GBAGUIDI"}]
+    let response = setBodyParameters(body, params)
+    let counter=0
+    params?.forEach(element => {
+      counter=counter+(response.match(/PARAMS_/g) || []).length
+    });
+    expect(counter).toBe(0)
+  })
+
+  test('Test setBodyParameters without passing parameters', async () => {
+    const body = "<h1>PARAMS_Lastname PARAMS_Firstname Send automated messages with Candymail</h1><p>Now with HTML support</p><a href='https://saasbase.dev/candymail'>Learn more here</a>"
```

Comment:
What do you think about: using `{}` vs. `PARAMS_`. The former option is used by most email automation software to represent replaceable values. 

For example - `{firstname}`

Reply:
- Yes at first i've thought the same way but, for email with code snipets and thing like that the end user of the package could probably write somethings like object `car={color:"red", brand:"ferrari", ...}` which could trigger errors at some point (i think). I don't know :sweat_smile: .

Reply:
That's definitely an extended use case - to be able to pass in object (the possibilities!). For this PR, let's assume it will only a string.

Reply:
Actually let's make it double curly braces - `{{firstname}}`

Reply:
Okey, i've made the change. It should be good now. Let's ship it 😍.